lib/modules/versioning/unity3d/index.ts

```diff
@@ -0,0 +1,57 @@
+import { regEx } from '../../../util/regex';
+import { GenericVersion, GenericVersioningApi } from '../generic';
+import type { VersioningApi } from '../types';
+
+export const id = 'unity3d';
+export const displayName = 'Unity3D';
+export const urls = [
+  'https://docs.unity3d.com/Manual/ScriptCompilationAssemblyDefinitionFiles.html#version-define-expressions',
+];
+export const supportsRanges = false;
+
+class Unity3dVersioningApi extends GenericVersioningApi {
+  private static readonly parsingRegex = regEx(
+    /^(?<Major>\d+)\.(?<Minor>\d+)\.(?<Patch>\d+)(?<ReleaseStream>\w)(?<Build>\d+)/gm,
```

Comment:
```suggestion
    /^(?<Major>\d+)\.(?<Minor>\d+)\.(?<Patch>\d+)(?<ReleaseStream>\w)(?<Build>\d+)/,
```
No multiline or multimatch necessary, this is probably also the reason why you have to reset the index.

Reply:
Makes sense! Addressed in 1d9e6f3fca09f8d02cdb7aa7e7d8496076592542.